cart-service/src/lib/commerce-tools-cart-mapper.test.ts

```diff
@@ -309,6 +310,16 @@ test("should not error if mapping configuration has nested path that doesn't exi
   expect(result.context?.test).toBeUndefined();
 });
 
+test("should use the default data instance if it's not set in the cart", async () => {
```

Comment:
This test is verifying the data instance is used from the cart, so I think a more appropriate name would be something like:

```suggestion
test("should use data instance from cart if set", async () => {
```

It would also be good to have a test that verifies the default value is `Live`. Guess can add that to one of the existing tests or create a new test.